### PR TITLE
fix(theme): Fixes icons sizes according to guidelines

### PR DESCRIPTION
**Pull request by talend-glorieux**
**Please check if the PR fulfills these requirements**

- [x] The commit(s) message(s) follows our
  [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What is the current behavior?** (You can also link to an open issue here)

Icons size doesn't follow the guideline.

**What is the new behavior?**

Icons size is conform to the guidelines.

### CODE REVIEW

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://148.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://148.talend.surge.sh)

File: packages/theme/src/theme/_guidelines.scss [hunk 1]
```diff
@@ -87,8 +87,8 @@ $svg-sm-size: 1.2rem !default;
 
 /// Default button svg size
 /// @type Number (Unitless)
-$svg-md-size: 2rem !default;
+$svg-md-size: 1.6rem !default;
 
 /// Large button svg size
 /// @type Number (Unitless)
-$svg-lg-size: 2.5rem !default;
+$svg-lg-size: 2.4rem !default;
```
Comment (jsomsanith) [inline packages/theme/src/theme/_guidelines.scss]: Shouldn't you add the `$svg-xlg-size: 3.2 rem !default;` defined in the guideline too?

Comment (talend-glorieux) [inline packages/theme/src/theme/_guidelines.scss]: I'm not gonna add it since it's not used.

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://148.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://148.talend.surge.sh)

**Pull request merged**